components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/triggerapi.go

```diff
@@ -136,14 +142,45 @@ func TriggerCheckAPI(endPoint, host, nodeType, method string, output chan<- mode
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusOK {
+		responseBody, err := io.ReadAll(resp.Body)
```

Comment:
instead of first comparing the resp.status, parse the resp body then write the logic for returning the correct result as per the resp body and resp code

Reply:
Discussed